src/private_pgn/tokenstream.js

```diff
@@ -41,18 +41,20 @@ var TokenStream = exports.TokenStream = function(pgnString, initialPosition) {
 	this._text           = pgnString;       // what is being parsed
 	this._pos            = initialPosition; // current position in the string
 	this._emptyLineFound = false;           // whether an empty line has been encountered while parsing the current token
+	this._lineCount      = 1;               // the current line number starting at 1
 	this._token          = 0;               // current token
 	this._tokenValue     = null;            // current token value (if any)
 	this._tokenIndex     = 0;               // position of the current token in the string
 
 	// Space-like matchers
 	this._matchSpaces = /[ \f\t\v]+/g;
 	this._matchLineBreak = /\r?\n|\r/g;
+	this._matchCommentLineBreak = /\r?\n|\r/g; // allow matching comment linebreaks separately
 
 	// Token matchers
 	this._matchHeaderRegular = /\[\s*(\w+)\s+"((?:[^\\"]|\\[\\"])*)"\s*\]/g;
 	this._matchHeaderDegenerated = /^\[\s*(\w+)\s+"(.*)"\s*\]$/mg;
```

Comment:
Adding support for castling moves encoded with `0` has been discussed in #6, and I'm not in favor of that. What is the origin of the corresponding PGN files?

Reply:
I'll find the pgn, I think it might actually be some old twic or Gorgobase pgn.